src/js/components/BreadcrumbComponent.jsx

```diff
@@ -76,8 +165,16 @@ var BreadcrumbComponent = React.createClass({
   },
 
   render: function () {
+    var classSet = classNames({
+      breadcrumb: true,
+      collapsed: this.state.collapsed
+    });
+
+    // Hacky, but ensures that the dimension measurements are up-to-date
+    _.defer(this.updateDimensions);
```

Comment:
Am I wrong or this is only needed in the first render? Otherwise it should be enough to call `this.updateDimensions` in `componentDidUpdate`, no?